### PR TITLE
Error page 403

### LINKED ISSUE #206: Nicer Error Message For People outside US.
Because of rights restrictions...content is fenced to within the US (https://github.com/NYPL-Simplified/web-reader-landing-page/issues/191):
It works! But its a default CloudFront error. **Let's investigate making it more humane.**

[Amazon's Documentation](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/georestrictions.html) has this example:

> Here's how geo restriction works:
> 
> Suppose you have rights to distribute your content only in Liechtenstein. You update your CloudFront web distribution and add a whitelist that contains only Liechtenstein. (Alternatively, you could add a blacklist that contains every country except Liechtenstein.)
> 
> A user in Monaco requests your content, and DNS routes the request to the CloudFront edge location in Milan, Italy.
> 
> The edge location in Milan looks up your distribution and determines that the user in Monaco is not allowed to download your content.
> 
> CloudFront returns an HTTP status code of 403 (Forbidden) to the user.
> 
> You can optionally configure CloudFront to return a custom error message to the user, and you can specify how long you want CloudFront to cache the error response for the requested object; the default value is five minutes. For more information, see Customizing Error Responses.

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes #206

### CODE REVIEW

File: 403.html.template [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        {{callOutText}}
+        {{downloadAppBtns}}
+      </div>
+    </nav>
+
+    <main>
+      <div class="error-page">
+        <p>We’re sorry…</p>
+        <p>Subway Library titles are not available in your country.</p>
+        <p>Visit the Subway Library <a href="https://subwaylibrary.com/">home page</a> or <a href="https://www.nypl.org/books-music-dvds/ebookcentral/simplye">learn more about SimplyE</a>.</p>
```
Comment (aslagle) [inline 403.html.template]: If someone is seeing this page they won't be able to visit the Subway Library home page...

**Comment by nodanaonlyzuul** [comment]
Yikes - yes - I accepted this PR already but @EdwinGuzman maybe we should just remove that line.
@jce1028 - any thoughts on places we should send users from outside the US? Just delete the link?

File: 403.html.template (hunk 1 above)
Comment (EdwinGuzman) [inline 403.html.template]: Is this error for all books outside of the US or just specific ones?

File: 403.html.template [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <meta property="og:description" content="Read free books, excerpts, and short stories from NYC public libraries." />
+  <meta property="og:url" content="https://www.subwaylibrary.com" />
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:site" content="@nypl" />
+  <meta name="twitter:creator" content="@nypl" />
+  <meta name="twitter:title" content="Subway Library" />
+  <meta name="twitter:description" content="Read free books, excerpts, and short stories from NYC public libraries." />
+  <meta name="twitter:image" content="https://www.subwaylibrary.com/twitter_share.jpg" />
+
+  <link rel="stylesheet" href="/index.min.css" />
```
Comment (aslagle) [inline 403.html.template]: I suspect the css file will also give a 403 error. This page might need to have all css and js embedded the way the call to action does.

Comment (nodanaonlyzuul) [inline 403.html.template]: I agree - sorry I hit accept on this so quickly folks.

File: 403.html.template (hunk 1 above)
Comment (aslagle) [inline 403.html.template]: I'm not sure but I thought the restriction was on everything in the S3 bucket.

Comment (nodanaonlyzuul) [inline 403.html.template]: This error will be served to all users who hit the site who aren't in the US, regardless of book.
🇺🇸  :eagle:

**Comment by EdwinGuzman** [comment]
I understood it as the website would be accessible, but not the books. I can make a different ticket for that and updated copy too.

File: 403.html.template (hunk 1 above)
Comment (courtneymcgee) [inline 403.html.template]: Sorry for the misunderstanding here. I thought non-US folks would be able to see home page, but get the error when they clicked on titles. Working on new copy...

**Comment by courtneymcgee** [comment]
New copy, @EdwinGuzman:

We’re sorry…
Subway Library is not available outside of the United States.

If @jce1028 has an idea about where to send non-US folks, we can add that in. But I can't think of anywhere we could send them that would be useful/helpful.

**Pull request merged**